examples/recorder/src/bot.ts

```diff
@@ -0,0 +1,46 @@
+import Discord, { Interaction } from 'discord.js';
+import { getVoiceConnection } from '@discordjs/voice';
+import { deploy } from './deploy';
+import { interactionHandlers } from './interactions';
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires, @typescript-eslint/no-require-imports
+const { token } = require('../auth.json');
+
+const client = new Discord.Client({ intents: ['GUILD_VOICE_STATES', 'GUILD_MESSAGES', 'GUILDS'] });
+
+client.on('ready', () => console.log('Ready!'));
+
+client.on('messageCreate', async (message) => {
+	if (!message.guild) return;
+	if (!client.application?.owner) await client.application?.fetch();
+
+	if (message.content.toLowerCase() === '!deploy' && message.author.id === client.application?.owner?.id) {
+		await deploy(message.guild);
+		await message.reply('Deployed!');
+	}
+});
```

Comment:
We might want to add a TODO and switch to slash-based deployments, since the event is becoming privileged.

Reply:
Can we not deploy slash commands in ready event ? by using `commands.set([<slash-commands>])`

It would be great alternative to that.